scripts/deploy/deployDCLControllerV2.js

```diff
@@ -0,0 +1,37 @@
+import ethProvider from 'eth-provider'
+import hre, { web3 } from 'hardhat'
+
+async function main() {
+  let chain
+  let mana
+  let dclRegistry
+  let feeCollector
+
+  switch (process.env.NETWORK) {
+    case 'GOERLI':
+      chain = 5
+      mana = '0xe7fDae84ACaba2A5Ba817B6E6D8A2d415DBFEdbe'
+      dclRegistry = '0x6b8da2752827cf926215b43bb8E46Fd7b9dDac35'
+      feeCollector = '0xE3336140Edfe740a26F9f1912fD52891e6Dd8A35'
+      break
+    default:
+      throw new Error('Invalid network')
+  }
+
+  const provider = ethProvider()
```

Comment:
This can be initialized as ethProvider('frame') but as the docs indicate:
<img width="861" alt="image" src="https://user-images.githubusercontent.com/24811313/225348993-f95aff83-fb67-4f0b-a0cc-e6d3737bfaf0.png">
'frame' is used by default